### PR TITLE
Update otel healthcheck port

### PR DESCRIPTION
**Pull request by jkandasa**
Signed-off-by: Jeeva Kandasamy <jkandasa@gmail.com>

### CODE REVIEW

**Comment by jkandasa** [comment]
@kevinearls Can you please review the changes?

**Comment by objectiser** [comment]
Thanks @jkandasa, but I will close this as the inverse should be true - the old hc port should only be used when using the otel images. I will create a PR shortly.

File: test/e2e/utils.go [hunk 1]
```diff
@@ -62,6 +62,8 @@ var (
 	vertxDelaySeconds    = int32(getIntEnv("VERTX_DELAY_SECONDS", 1))
 	vertxTimeoutSeconds  = int32(getIntEnv("VERTX_TIMEOUT_SECONDS", 1))
 
+	otelHcPort = "13133" // Opentelemetry healthcheck port
```
Comment (jpkrohling) [inline test/e2e/utils.go:65]:
```suggestion
	otelHCPort = "13133" // OpenTelemetry health check port
```

**Comment by jpkrohling** [comment]
Indeed, I missed that!

**Pull request closed without merging**